src/components/Registration/BusinessNumber.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div id="business-number">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block">Business Number</label>
+        <v-chip
+          v-if="hasBusinessNumberChanged"
+          id="changed-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Changed
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0">
+          If you have an existing business number, enter it below and we will contact
+          Canada Revenue Agency and ask them to link it to this registration.
+        </p>
+        <HelpBusinessNumber class="mt-4" />
+        <v-text-field
+          filled
+          persistent-hint
+          ref="textField"
+          class="item mt-4 mb-n2"
+          label="Business Number (Optional)"
+          hint="First 9 digits of the CRA Business Number"
+          v-model="value"
+          :error="!valid"
+          :error-messages="errorMessages"
+          @blur="onBlur()"
```

Comment:
Note that I am not using rules here. Instead, I am using explicit `error` state and `error-messages` and custom validation logic below.

In short, I had to do this because of the desired validation behaviour, because there is no form "Done" button, and because calling validate() on this component doesn't work as desired due to its internal state (eg, whether it's touched or dirty).